src/ApiProblem.php

```diff
@@ -215,20 +215,28 @@ protected static function decompile(array $parsed) : self
     {
         $problem = new static();
 
-
-        if (!empty($parsed['title'])) {
+        // Skip empty string or missing values.
+        // The string or integer 0, however, are allowed.  PHP makes
+        // this ugly. The check on string handles XML decompile which
+        // may return an empty array.
+        $emptyValues = ['', []];
+        if (isset($parsed['title']) && !in_array($parsed['title'], $emptyValues, true)) {
```

Comment:
This check seems like something that should be factored out to a utility method.  It's basically a more forgiving `is_empty()`.  It's used repeatedly, though, so let's make it its own method and move the comment describing it there.

Suggested name: `validValue(string $str): bool` (so it ends up reading `if(static::validValue($parsed['title']){ ... }`